### PR TITLE
[Fix] correct bridge env vars names for consumer and producer enabled

### PR DESCRIPTION
**Pull request by kornys**
### Type of change

_Select the type of your PR_

- Bugfix

### Description

We can rename env vars there or in bridge. please see this PR
related bridge issue https://github.com/strimzi/strimzi-kafka-bridge/pull/900

### CODE REVIEW

**Comment by scholzj** [comment]
Its great that you noticed this issue. Thanks. But maybe its best to just start a discussion on Slack and see what @ppatierno as the SME thinks about it rather than madly opening PRs all over the place.

**Comment by kornys** [comment]
> Its great that you noticed this issue. Thanks. But maybe its best to first start a discussion on Slack and see what @ppatierno as the SME thinks about it rather than madly opening conflicting PRs all over the place.

Well I wanted to help and save time so I have opened PR in both repositories. SME can just pick what is better solution and close the second one. So I don't see any issue with this approach...

**Review by Frawless — APPROVED** [review]

**Review by im-konge — APPROVED** [review]

File: cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java [hunk 1]
```diff
@@ -111,8 +111,8 @@ public class KafkaBridgeCluster extends AbstractModel implements SupportsLogging
     protected static final String ENV_VAR_KAFKA_BRIDGE_HTTP_HOST = "KAFKA_BRIDGE_HTTP_HOST";
     protected static final String ENV_VAR_KAFKA_BRIDGE_HTTP_PORT = "KAFKA_BRIDGE_HTTP_PORT";
     protected static final String ENV_VAR_KAFKA_BRIDGE_HTTP_CONSUMER_TIMEOUT = "KAFKA_BRIDGE_HTTP_CONSUMER_TIMEOUT";
-    protected static final String ENV_VAR_KAFKA_BRIDGE_CONSUMER_ENABLED = "KAFKA_BRIDGE_CONSUMER_ENABLED";
-    protected static final String ENV_VAR_KAFKA_BRIDGE_PRODUCER_ENABLED = "KAFKA_BRIDGE_PRODUCER_ENABLED";
+    protected static final String ENV_VAR_KAFKA_BRIDGE_CONSUMER_ENABLED = "KAFKA_BRIDGE_HTTP_CONSUMER_ENABLED";
+    protected static final String ENV_VAR_KAFKA_BRIDGE_PRODUCER_ENABLED = "KAFKA_BRIDGE_HTTP_PRODUCER_ENABLED";
```
Comment (im-konge) [inline cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java:115]:
```suggestion
    protected static final String ENV_VAR_KAFKA_BRIDGE_HTTP_CONSUMER_ENABLED = "KAFKA_BRIDGE_HTTP_CONSUMER_ENABLED";
    protected static final String ENV_VAR_KAFKA_BRIDGE_HTTP_PRODUCER_ENABLED = "KAFKA_BRIDGE_HTTP_PRODUCER_ENABLED";
```
I would rename the variables as well, to follow the style there

**Comment by ppatierno** [comment]
I prefer this way, so fixing in the operator.

File: cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java (hunk 1 above)
Comment (ppatierno) [reply cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java:115]: +1

**Review by ppatierno — APPROVED** [review]
LGTM (after fixing the comment from Lukas)

**Review by im-konge — APPROVED** [review]

**Pull request merged**